samples/cellular/at_client/prj.conf

```diff
@@ -23,3 +23,5 @@ CONFIG_UART_INTERRUPT_DRIVEN=y
 # Stacks and heaps
 CONFIG_MAIN_STACK_SIZE=3072
 CONFIG_HEAP_MEM_POOL_SIZE=16384
+CONFIG_NRF_MODEM_LIB_TRACE=y
```

Comment:
Traces should be enabled with the `nrf91-modem-trace-uart` snippet. This enables the deprecated UART trace backend.

Reply:
Should the default behavior of this sample be incompatible with Cellular Monitor?
Or should we add the contents of the snippet directly?

Reply:
I changed it to using the new backend now.

Reply:
I agree that we should keep the traces disabled by default and document how to enable them. Ideally we might want to do this for other samples as well to keep it consistent.

Reply:
Alright, I changed the PR to only include the heap size option. Still, I think this will be confusing for customers.